plugin/trino-hive/src/test/java/io/trino/plugin/hive/metastore/glue/TestHiveGlueMetastore.java

```diff
@@ -152,34 +154,59 @@ public void testRenameTable()
         // rename table is not yet supported by Glue
     }
 
-    @Override
-    public void testPartitionStatisticsSampling()
-    {
-        // Glue metastore does not support column level statistics
-    }
-
     @Override
     public void testUpdateTableColumnStatistics()
+            throws Exception
     {
-        // column statistics are not supported by Glue
+        // The original implementation AbstractTestHiveClient#testUpdateTableColumnStatistics assumes each update call to metastore would override
+        // the previous update call but this is not entirely true for Glue.
+        // For example, there are 5 columns of a table. if you update 2 columns and then update the remaining 3 columns in two separate calls.
+        // The second update call would not override the first update call (assume the first 2 and remaining 3 columns are exclusive).
+        // Following this rationale, provide an empty column stat does not clear all the existing column stats.
+        // (Glue actually does not accept the empty column stat).
+        SchemaTableName tableName = temporaryTable("update_table_column_statistics");
+        try {
+            doCreateEmptyTable(tableName, ORC, STATISTICS_TABLE_COLUMNS);
+            // STATISTICS_1_1 must be subset of STATISTICS_1
+            testUpdateTableStatistics(tableName, EMPTY_TABLE_STATISTICS, STATISTICS_1_1, STATISTICS_1);
+        }
+        finally {
+            dropTable(tableName);
+        }
     }
 
     @Override
     public void testUpdateTableColumnStatisticsEmptyOptionalFields()
+            throws Exception
     {
-        // column statistics are not supported by Glue
+        // this test is not really meaningful for Glue, for numeric columns, min/max stat should always be available
+        // The only exception is when all rows are NULLs. Glue requires min/max to be set.
     }
 
     @Override
-    public void testUpdatePartitionColumnStatistics()
+    public void testUpdatePartitionColumnStatisticsEmptyOptionalFields()
+            throws Exception
     {
-        // column statistics are not supported by Glue
+        // this test is not really meaningful for Glue, for numeric columns, min/max stat should always be available
+        // The only exception is when all rows are NULLs. Glue requires min/max to be set.
```

Comment:
`throw new SkipException` would be more appropriate, but i'd still want to understand better what is supposed to be tested here and cannot be

Reply:
This is kinda strange, from my understanding here the test try to perform a statistics update for all the columns but without min/max values.
Min / Max values are actually optional according to the [docs](https://docs.aws.amazon.com/glue/latest/webapi/API_LongColumnStatisticsData.html) .
But the Glue api answer with an `Status Code: 500; Error Code: InternalServiceException` Without giving much context. 
Requests were all the statistics (min, max, nulls ...) are specified are completing correctly

Reply:
@GaruGaru knowing the deficiency of Glue can we make the code write minimal/maximal value for a type if actual min/max is not present in `HiveColumnStatistics`?
So for BIGINT we would write `-2^63`/`2^63 - 1` for INTEGER `-2^31``/`2^31 - 1` ... and so on.

Reply:
@GaruGaru has this one been addressed?